### ISSUE TITLE
terraform plan got error because local chart not rendered yet

### ISSUE THREAD

**Issue by shuliakovsky** [hashicorp/terraform-provider-helm#1348]
### Terraform, Provider, Kubernetes and Helm Versions

```
Terraform version: v1.7.5
Provider version: v2.13.0
Kubernetes version: 1.29
```

### Affected Resource(s)

- helm_release

### Terraform Configuration Files
```hcl

# rendering chart
resource "local_file" "this" {
  count    = length(local.template_files) # Define right template
  filename = "${path.module}/${var.eks_cluster_name}/helm/${var.project}/${local.template_files[count.index]}"
  content  = templatefile("${path.module}/templates/${local.template_files[count.index]}", local.helmChartVariables)
}
resource "time_sleep" "wait_15_seconds" {
  depends_on      = [local_file.this]
  create_duration = "15s"
  triggers = {
    always_run = timestamp() # Always run
  }
}
# Deploy rendered helm-chart
resource "helm_release" "this" {
  name      = var.project
  chart     = "${path.module}/${var.eks_cluster_name}/helm/${var.project}"
  namespace = var.environment
  wait      = true
  timeout   = 600
  depends_on = [
    local_file.this,
    time_sleep.wait_15_seconds,
  ]
}
```

### Debug Output

NOTE: In addition to Terraform debugging, please set HELM_DEBUG=1 to enable debugging info from helm.

### Panic Output

non
### Steps to Reproduce

1. `terraform plan`

### Expected Behavior
plan working without error

### Actual Behavior
Error: path "../../" not found

### References

Fix manifest diffs for OCI charts (https://github.com/hashicorp/terraform-provider-helm/pull/1326)

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by shuliakovsky** [comment]
https://github.com/hashicorp/terraform-provider-helm/commit/d5337341a5b127f9d0ea223590fd1c2e580e39b7#comments
link where error is happening

**Comment by shuliakovsky** [comment]
temporary solution is: locking provider version on any previous one which fits you i.e.
```
terraform {
  required_providers {
    helm = {
      version = "2.12.1"
      source  = "hashicorp/helm"
    }
  }
```

**Comment by jrhouston** [comment]
Thanks for opening this @shuliakovsky. This looks like a regression, I'm going to revert [the change](https://github.com/hashicorp/terraform-provider-helm/commit/d5337341a5b127f9d0ea223590fd1c2e580e39b7#comments) mentioned above.

**Comment by MrImpossibru** [comment]
I’m getting an error about “repo build not found” during the plan stage with the new provider version. Looks like almost the same problem.

**Comment by shuliakovsky** [comment]
fixed in v2.13.1
